src/Common/src/CoreLib/System/Diagnostics/Tracing/TraceLogging/Statics.cs

```diff
@@ -174,7 +174,7 @@ public static void EncodeTags(int tags, ref int pos, byte[] metadata)
 
         public static void CheckName(string name)
         {
-            if (name != null && 0 <= name.IndexOf('\0'))
```

Comment:
FYI, you don't need to make the changes to src/Common/src/CoreLib/* again here.
The changes from https://github.com/dotnet/coreclr/pull/19874 will get mirrored to this repository automatically.

Reply:
See https://github.com/dotnet/corefx/pull/32252

Reply:
Reverted those files